source/content.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  "home": {
+    "url": "index.html",
+    "content": "home.html",
+    "stylesheets": ["/home.css"]
+  },
+  "es6": {
+    "url": "es6.html"
+  },
+  "faq": {
+    "url": "faq.html"
+  }
+}
```

Comment:
@therebelrobot and I had a discussion around this in some other thread. Basically I expect some of these files to get very complex and, in general, we thought we'd try `.js` to start to we get a little more flexibility on formatting these things as they grow.

I'll try and link you to the other conversation.